app_dart/lib/src/service/config.dart

```diff
@@ -57,6 +57,15 @@ class Config {
         pluginsSlug,
       };
 
+  /// List of Github repos that are mirrored to GoB.
+  Set<gh.RepositorySlug> get mirroredRepos => <gh.RepositorySlug>{
```

Comment:
Should this be removed? Per the security policy, all repos running on LUCI should be mirrored.

Reply:
Sounds good. I've replaced this with logic in `isGithubRepoMirroredToGob` that checks if the slug is present in either `supportedRepos` or `postsubmitSupportedRepos`.